README.md

```diff
@@ -20,9 +20,9 @@ accordingly.
 ## Overview
 
 This repository is the home of the core Python rules -- `py_library`,
-`py_binary`, `py_test`, and related symbols that provide the basis for Python
-support in Bazel. It also contains packaging rules for integrating with PyPI
-(`pip`). Documentation lives in the
+`py_binary`, `py_test`, `py_import`, and related symbols that provide the
```

Comment:
I don't think this list is meant to be exhaustive, plus it's clearly very insufficient for documenting the rules. I'm looking at generating a better docsite similar to the https://github.com/bazelbuild/rules_nodejs one.
I'll revert this file to keep the delta small